docs/deployment.md

```diff
@@ -125,6 +125,8 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
             -e NODE_ENV=production \
             -e PRIVATE_KEY="$(cat ~/Downloads/*.private-key.pem)"
 
+      **NOTE**: Add `-e LOG_LEVEL=trace` to get verbose logging,or use 'info' instead of trace to show less details.
```

Comment:
I think, showing both the examples would be better like `-e LOG_LEVEL=trace` for detailed and `-e LOG_LEVEL=info` for less detailed log